com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -23,72 +23,24 @@ namespace UnityEngine.Rendering.HighDefinition
     /// </summary>
     public partial class HDRenderPipeline : RenderPipeline
     {
-        #region Default Settings
-        internal static HDRenderPipelineAsset defaultAsset
-            => GraphicsSettings.renderPipelineAsset is HDRenderPipelineAsset hdrpAsset ? hdrpAsset : null;
+        #region Global Settings
+        internal static HDRenderPipelineGlobalSettings defaultAsset
```

Comment:
Shouldn't we call this globalSettings now? 
Same for the override bellow? Maybe this one cannot be change now since it's C++ though?

Reply:
for the defaultAsset: i am not changing it right now to not break people's code. but i could make it obsolete in a subsequent PR?

Reply:
The API is internal so it does not need to go through the whole obsolescence process. No one should have been able to use it so it should be safe.

Reply:
I have added this as a polish task after that PR if that is ok